docs/docs/usage/1-memory.md

```diff
@@ -54,8 +57,9 @@ int ptr = (int) alloc.apply(len)[0];
 memory.writeString(ptr, message);
 ```
 
-Now we can call `countVowels` with this pointer to the string. It will do its job and return the count. We will
-call `dealloc` to free that memory in the module. Though the module could do this itself if you want:
+Now we can call `countVowels` with this pointer to the string.
+It will do its job and return the count.
+We will call `dealloc` to free that memory in the module. Though the module could do this itself if you want:
```

Comment:
I removed the sentence, as I think it confuses the issue. We could add another paragraph below to cover this topic if you think it's helpful. Having the caller free the memory seems much more common than the other model (and matches the C standard library), so I'm not sure we should mention it.

```suggestion
We will call `dealloc` to free that memory in the module:
```

Reply:
I like your wording :+1: